source/PlayerInfo.cpp

```diff
@@ -2550,14 +2550,13 @@ void PlayerInfo::ValidateLoad()
 
 	// Validate the missions that were loaded. Active-but-invalid missions are removed from
 	// the standard mission list, effectively pausing them until necessary data is restored.
-	missions.sort([](const Mission &lhs, const Mission &rhs) noexcept -> bool { return lhs.IsValid(); });
-	auto isInvalidMission = [](const Mission &m) noexcept -> bool { return !m.IsValid(); };
-	auto mit = find_if(missions.begin(), missions.end(), isInvalidMission);
+	auto mit = partition(missions.begin(), missions.end(), [](const Mission &m) { return m.IsValid(); });
```

Comment:
If we do need this change, can we use [`std::mem_fn`](https://en.cppreference.com/w/cpp/utility/functional/mem_fn) instead of writing the same thing?